runtime/world.js

```diff
@@ -227,23 +227,16 @@ module.exports = function () {
 
                 scenario.attach(new Buffer(screenShot, 'base64'), 'image/png');
 
-                return driver.close().then(function () {
-                    return driver.quit();
-                })
-                .then(function() {
-
+                return driver.close().then(function() {
```

Comment:
What about none geko browsers? It might be best to add a condition to skip `.quit()` for geko browser?